packages/loopring_v3/circuit/Utils/Data.h

```diff
@@ -104,6 +104,7 @@ static auto dummyDeposit = R"({
     "amount": "0"
 })"_json;
 
+// QUESTION(daniel): how `s` is calculated in "dummySignature"?
```

Comment:
`s` is just another part of the signature, so I guess similar to how `Rx` and `Ry` are calculated, though that doesn't answer a lot of questions. :)

https://github.com/Loopring/protocols/blob/9898bf2bdf4966d021eb9b2745fab00799b35a94/packages/loopring_v3.js/src/eddsa.ts#L106

Reply:
I think we should use a known seed for generating these dummy signatures/keys, something like 'LOOPRING ZKROLLUP'

Reply:
Changed it to the base point, which is even easier to verify: https://github.com/Loopring/protocols/pull/1572/commits/50efb5df006405e4b405dab6bd6fd091ff71ff53